x-pack/plugins/alerts/common/index.ts

```diff
@@ -22,3 +22,12 @@ export interface AlertingFrameworkHealth {
 
 export const BASE_ALERT_API_PATH = '/api/alerts';
 export const ALERTS_FEATURE_ID = 'alerts';
+
+export enum LicenseTypeValues {
+  Basic = 'basic',
+  Gold = 'gold',
+  Platinum = 'platinum',
+  Enterprise = 'enterprise',
+  Standard = 'standard',
+  Trial = 'trial',
+}
```

Comment:
It seems like a duplication of sources and similar to `LICENSE_TYPE` in the licensing plugin?